app/controllers/identified_clients_controller.rb

```diff
@@ -23,20 +23,14 @@ def destroy
     respond_with(@non_hmis_client, location: identified_clients_path)
   end
 
+  def assessment_type
+    Config.get(:identified_client_assessment) || 'IdentifiedClientAssessment'
```

Comment:
This will need to hit a whitelist.  Probably something on `NonHmisAssessment` like.
`def self.assessment_type(type)
    {
      'NonHmisAssessment' => NonHmisAssessment,
      'IdentifiedClientAssessment' => IdentifiedClientAssessment,
      'IdentifiedPathwaysAssessment' => IdentifiedPathwaysAssessment,
    }.freeze[type] || NonHmisAssessment
  end`